ibm/mas_devops/roles/cp4d_service/defaults/main.yml

```diff
@@ -61,6 +61,10 @@ cpd_wsl_project_name: "{{ lookup('env', 'CPD_WSL_PROJECT_NAME') | default('ws_de
 cpd_wsl_project_description: "{{ lookup('env', 'CPD_WSL_PROJECT_DESCRIPTION') | default('Watson Studio Project for Maximo Application Suite', true) }}"
 cpd_wsl_project_storage_guid: "{{ 1000 | random | to_uuid | lower }}"
 
+# CPD Watson Discovery default properties
+# ---------------------------------------------------------------------------------------------------------------------
+cpd_wd_instance_name: discovery-assist
```

Comment:
Should this not be configurable, or at minimum contain the mas instance id / mas workspace id to avoid collisions with multi-workspace and/or multi-instance installs?